scripts/runmariadb

```diff
@@ -17,6 +17,7 @@ if [ ! -f "$MARIADB_CERT_FILE" ] && [ -f "$MARIADB_KEY_FILE" ] ; then
 fi
 
 ln -sf /proc/self/fd/1 /var/log/mariadb/mariadb.log
```

Comment:
Can we maybe use some normal approach to logging rather than this hack? https://mariadb.com/kb/en/error-log/#writing-the-error-log-to-stderr-on-unix suggests it should be possible to just log to stderr.